test-integration/helpers/course-select.coffee

```diff
@@ -1,17 +1,37 @@
 selenium = require 'selenium-webdriver'
 {TestHelper} = require './test-element'
 
+
+COMMON_ELEMENTS =
+  courseLink: (appearance, isCoach = false) ->
+    dataAttr = 'data-appearance'
+
+    if appearance?
+      dataAttr += "='#{appearance}'"
+
+    if isCoach
+      teacherLink = "[#{dataAttr}] > [href*='cc-dashboard']"
```

Comment:
Okay, we can look at this for a follow-up PR.  This pr has quite a few updates to the base `TestHelper` that would be useful in other PRs